.kitchen.yml

```diff
@@ -1,78 +1,37 @@
 ---
 driver:
-  name: dokken
+  name: vagrant
```

Comment:
I couldn't run `dokken` on my windows machine. Plus I'm not convinced this ever worked, at least for the chef server. So it was for my testing. I'd be happy to roll it back, but don't see `dokken` as adding any benefit.